i18n/zh/docusaurus-plugin-content-docs/current/installation/installation.md

```diff
@@ -1,42 +1,38 @@
 ---
-title: Installation Overview
+title: 安装概述
 ---
 
-## Prerequisites
+## 前提条件
 
-### Karmada kubectl plugin
-`kubectl-karmada` is the Karmada command-line tool that lets you control the Karmada control plane, it presents as
-the [kubectl plugin][1].
-For installation instructions see [installing kubectl-karmada](./install-kubectl-karmada.md).
+### Karmada kubectl 插件
+`kubectl-karmada` 是允许你控制 Karmada 控制面的 Karmada 命令行工具，该工具表现为一个 [kubectl 插件][1]。
+有关安装指导，请参见[安装 kubectl-karmada](./install-kubectl-karmada.md)。
 
 ### Karmadactl
-`karmadactl` is also the command-line tool that lets you control the Karmada control plane. Compared with kubectl-karmada,
-it is a complete cli tool exclusive to Karmada. 
-For installation instructions see [installing karmadactl](./install-karmadactl.md).
-
+`karmadactl` 也是允许你控制 Karmada 控制面的 Karmada 命令行工具。
+与 kubectl-karmada 相比，`karmadactl` 是一个完全专用于 Karmada 的 CLI 工具。
+有关安装指导，请参见[安装 karmadactl](./install-karmadactl.md)。
 :::note
 
-Although the above two tools have different forms, the related commands and options are exactly the same. 
-The following takes kubectl-karmada as an example.
-Replacing it with karmadactl also works fine.
+尽管以上两个工具的名字不同，但其关联的命令和选项完全相同。
+这里以 kubectl-karmada 为例。如果替换为 karmadactl，也能工作得很好。
```

Comment:
#205 add a node "In actual use, you can choose a CLI tool according to your needs." Could you add it in Chinese too?

Reply:
ok, noted and changed after rebase 😃